test/axes/timeAxisTests.ts

```diff
@@ -303,6 +303,24 @@ describe("TimeAxis", () => {
         assert.strictEqual(annotationFormatter(testDate), "Sun Dec 17, 1995", "formats to a default customized time formatter");
       });
     });
+
+    it("includes the annotation space in the final size calculation", () => {
+      let svg = TestMethods.generateSVG(400, 400);
+      let xScale = new Plottable.Scales.Time();
+      let xAxis = new Plottable.Axes.Time(xScale, "bottom");
+      xAxis.margin(100);
+      xAxis.annotationsEnabled(true);
+      xAxis.annotationTierCount(3);
+
+      xAxis.anchor(svg);
+      xAxis.computeLayout({ x: 0, y: 0}, 400, 400);
+      let coreHeight = xAxis.tickLabelPadding() + xAxis.innerTickLength();
+      let annotationHeight = xAxis.annotationTierCount() * (<any> xAxis)._annotationTierHeight();
```

Comment:
Not sure how seeing if the three annotations overlap really does much.  The main issue is that the visibility of the rectangles is hidden due to bad space calculations.